backend/backend/urls.py

```diff
@@ -0,0 +1,25 @@
+"""
+URL configuration for transcendence project.
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/5.0/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path, include
+from . import views
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
+	path('json/', views.json, name='json'),
```

Comment:
ideally consistent approach using include (as done for pong); include probably also provides more flexibility if something changes in the usermanagement.urls

Also, this is for testing, right? (otherwise needs a more descriptive name)